spec/lib/traject/config/format_physical_spec.rb

```diff
@@ -370,12 +370,6 @@
       MARC::Record.new.tap do |r|
         r.leader = '02229cjm a2200409Ia 4500'
         r.append(MARC::ControlField.new('007', 'sd fungnnmmneu'))
-        r.append(MARC::DataField.new('999', ' ', ' ',
```

Comment:
We could probably delete this whole test not just the 999

Reply:
I updated the context instead to show why it is relevant to the code.

Reply:
@jcoyne I was looking into the following `it` condition here, the whole test being:

```
  context 'when acccess_facet is not "At the library" (based on 8833535)' do
    let(:record) do
      MARC::Record.new.tap do |r|
        r.leader = '02229cjm a2200409Ia 4500'
        r.append(MARC::ControlField.new('007', 'sd fungnnmmneu'))
      end
    end

    it 'is not a CD, since it is online-only' do
      expect(result[field]).to eq nil
    end
  end
```

I'm not sure the "since it is online-only' makes sense here. The access_facet is now just empty. Should it read "it is not a CD, since it is not at the library"? Does this even make sense?

Reply:
Okay I've just updated to say what they are (nil) instead of what they are not.

Reply:
Nice that makes more sense to me!